source/main.cpp

```diff
@@ -247,10 +249,12 @@ void GameLoop(PlayerInfo &player, Conversation &conversation, bool &debugMode)
 			showCursor = shouldShowCursor;
 			SDL_ShowCursor(showCursor);
 		}
-		// Switch off fast-forward if the player is not in flight (for example when the
-		// boarding dialog shows up or when the player lands or opens the map). The player
-		// can switch fast-forward on again when flight is resumed.
-		if(!inFlight && isFastForward)
+		bool inMap = !gamePanels.IsEmpty() && (dynamic_cast<MapPanel*>(gamePanels.Top().get()) != nullptr);
+		bool inPlayerInfo = !gamePanels.IsEmpty() && (dynamic_cast<PlayerInfoPanel*>(gamePanels.Top().get()) != nullptr);
+		// Switch off fast-forward if the player is not in flight or flight-related screen
+		// (for example when the boarding dialog shows up or when the player lands or opens
+		// the map). The player can switch fast-forward on again when flight is resumed.
```

Comment:
Opening the map is now an exception, so should be removed from this list of examples of where FFW is switched off.

Reply:
Agreed, updated the comment.